### ISSUE TITLE
[BUG] The header of Alerting monitor details flyout is not aligned with the page tabs height

### ISSUE THREAD

**Issue by xeniatup** [opensearch-project/alerting-dashboards-plugin#298]
**What is the bug?**
The header of the monitor details flyout has the same type of underline as the tabs on the page. The two lines almost meet at the edge of the flyout, but the difference in the positions of the lines is too small to look intentional, which brings an appearance of a bug.

**How can one reproduce the bug?**

- Go to `Alerting`
- Click on any entry in the `Alerts by trigger` table (assuming that the table is populated)
- See the flyout with monitor details

**What is the expected behavior?**
The lines under the page tabs and the flyout title should either be on the same level or farer apart for the page to look organized in a more structured way.

**What is your host/environment?**

- OS: macOS
- Version 12.4
- Plugins Alerting
- Browser Chrome, Firefox, Safari

**Do you have any screenshots?**

<img width="1507" alt="Screen Shot 2022-08-04 at 12 13 51 AM" src="https://user-images.githubusercontent.com/105245012/182788707-5bc92cf2-8ee7-4af9-8516-b8508812e4c0.png">

**Comment by AWSHurneyt** [comment]
Good catch. If the lines can't be the same, I'll make the flyout header a little bigger so the difference looks more intentional.

**Comment by amsiglan** [comment]
No longer relevant as we moved to using overlay type of flyouts.
